### PR TITLE
Command line flag to print symbolic states as dot files

### PR DESCRIPTION
**Pull request by benozol**

### CODE REVIEW

File: lib/constraints/src/common/conj.ml [hunk 1]
```diff
@@ -58,7 +58,7 @@ let pp_as_dot ~name fmt (es, c) =
 
   let hash = Hashtbl.hash in
 
-  fpf fmt "@[<h 2>digraph %s {@\n" name;
+  fpf fmt "@[<h 2>digraph %S {@\n" name;
```
Comment (Niols) [inline lib/constraints/src/common/conj.ml]: What does it change?

File: src/colis_cmd.ml [hunk 2]
```diff
@@ -69,14 +69,15 @@ let speclist =
     "--symbolic-fs",               String set_symbolic_fs,        " Name of the initial symbolic filesystem in symbolic execution (values: empty, simple, fhs, default: empty)";
```
Comment (Niols) [inline src/colis_cmd.ml]: Shouldn't that be `"FS Name of the ..."`?

File: lib/constraints/src/common/conj.ml (hunk 1 above)
Comment (benozol) [reply lib/constraints/src/common/conj.ml]: S: convert a string argument to OCaml syntax (double quotes, escapes).

Graphviz wasn’t happy with the dashes in the graph name.

File: src/colis_cmd.ml (hunk 2 above)
Comment (benozol) [reply src/colis_cmd.ml]: Alright!

File: lib/constraints/src/common/conj.ml (hunk 1 above)
Comment (Niols) [reply lib/constraints/src/common/conj.ml]: Oh, I didn't know that. Nice :-)

**Comment by benozol** [comment]
Good to go?

**Comment by Niols** [comment]
Yes, I think so.

**Pull request merged**